### PR TITLE
Blog post about the new release

### PR DESCRIPTION
**Pull request by zner0L**

### CODE REVIEW

File: content/en/blog/v2-release/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "tags": [ "complaint generator", "new user experience", "update", "open source" ],
+    "authors": [ "zner0L" ]
+}
+
+In an old Christian tradition, German children get presents in their shoes on the morning of the 6th of December. Today, we put something in your shoes: The new release of our websites with many great new features! We spent a lot of time in the last few months to update the user interface and took great care to make it as easy and accessible as possible to send requests to comapnies and other data controllers. Thanks to the [Prototype Fund](https://prototypefund.de/en/) run by the [Open Knowledge Foundation Germany](https://okfn.de/en/) and funded by the [German Federal Ministry of Education and Research](https://www.bmbf.de/bmbf/en/home/home_node.html), who gave a grant to Lorenz and Benni, we were able to put more time than usual into this release (read more in our {{< link slug="verein/transparency#12-external-grants-that-members-have-received" text="transparency information" >}}). Now we are just really excited and proud to present you all the new features. If you want to take a look at the code, you can [find all changes on GitHub](TODO: Link to release).
```
Comment (mal-tee) [inline content/en/blog/v2-release/index.md:11]:
```suggestion
In an old Christian tradition, German children get presents in their shoes on the morning of the 6th of December. Today, we put something in your shoes: The new release of our websites with many great new features! We spent a lot of time in the last few months to update the user interface and took great care to make it as easy and accessible as possible to send requests to companies and other data controllers. Thanks to the [Prototype Fund](https://prototypefund.de/en/) run by the [Open Knowledge Foundation Germany](https://okfn.de/en/) and funded by the [German Federal Ministry of Education and Research](https://www.bmbf.de/bmbf/en/home/home_node.html), who gave a grant to Lorenz and Benni, we were able to put more time than usual into this release (read more in our {{< link slug="verein/transparency#12-external-grants-that-members-have-received" text="transparency information" >}}). Now we are just really excited and proud to present you all the new features. If you want to take a look at the code, you can [find all changes on GitHub](TODO: Link to release).
```

File: content/en/blog/v2-release/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+    "title": "Datenanfragen v2: Refreshed interface",
```
Comment (baltpeter) [inline content/en/blog/v2-release/index.md:2]:
```suggestion
    "title": "datarequests.org v2: Refreshed interface and complaint generator",
```

File: content/en/blog/v2-release/index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Comprehensive request history
+
+If you use our website regularly, you might be familiar with the “My requests” page. If you didn't disable it in the {{< link slug="/privacy-controls" text="privacy controls" >}}, we save all the requests you sent in your browser and give you an overview of them. Up until now, this was just a pretty convoluted table, but for years we wanted to improve upon it and rebuild it as a proper request management page. In our new release, we create a proceeding for each of your requests which holds all messages you sent relating to it, e.g. admonitions and complaints. You can add information about messages you received from the controller, so you have it all in one place once you want to complain. We save a state for all requests, so you know your current progress, and we remind you with a badge on all pages if there are requests overdue you should send an admonition for.
```
Comment (baltpeter) [inline content/en/blog/v2-release/index.md:19]:
```suggestion
If you use our website regularly, you might be familiar with the “My requests” page. If you didn't disable it in the {{< link slug="/privacy-controls" text="privacy controls" >}}, we save all the requests you sent in your browser and give you an overview of them. Up until now, this was just a pretty convoluted table, but for years we wanted to improve upon it and rebuild it as a proper request management page. In our new release, we create a proceeding for each of your requests which holds all messages you sent relating to it, including admonitions and complaints. You can add information about messages you received from the controller, so you have it all in one place if you need to send a complaint. We save the state of all requests, so you know your current progress, and we remind you with a badge on all pages if there are requests overdue you should send an admonition for.  
```

**Review by baltpeter — CHANGES_REQUESTED** [review]

File: content/en/blog/v2-release/index.md (hunk 1 above)
Comment (baltpeter) [reply content/en/blog/v2-release/index.md:11]: comapnies -> companies

Not sure why that isn't being highlighted.

File: content/de/blog/v2-release/index.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+{
+    "title": "Datenanfragen.de v2: Nagelneue Oberfläche und Beschwerdegenerator",
```
Comment (baltpeter) [inline content/de/blog/v2-release/index.md:2]: "v2" feels more English to me. I'd say, in German you'd more commonly say "2.0".

```suggestion
    "title": "Datenanfragen.de 2.0: Nagelneue Oberfläche und Beschwerdegenerator",
```

File: content/de/blog/v2-release/index.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    "type": "blog",
+    "date": "2022-12-07T12:00:00+02:00",
+    "description": "Gestern haben wir die neue Version der Webseite mit vielen neuen Features veröffentlicht: Eine neue Oberfläche für den Anfragengenerator, eine umfangreichere Anfragenhistorie, eine schicke neue Startseite und ein Generator für Beschwerden und Mahnungen.",
+    "featured_image": "stockings-feature",
+    "tags": [ "Beschwerdegenerator", "Neue Nutzer_innenoberfläche", "Update", "Open Source" ],
```
Comment (baltpeter) [inline content/de/blog/v2-release/index.md:7]:
```suggestion
    "tags": [ "Beschwerdegenerator", "Neue Oberfläche", "Update", "Open Source" ],
```

File: content/de/blog/v2-release/index.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
+{
+    "title": "Datenanfragen.de v2: Nagelneue Oberfläche und Beschwerdegenerator",
+    "type": "blog",
+    "date": "2022-12-07T12:00:00+02:00",
+    "description": "Gestern haben wir die neue Version der Webseite mit vielen neuen Features veröffentlicht: Eine neue Oberfläche für den Anfragengenerator, eine umfangreichere Anfragenhistorie, eine schicke neue Startseite und ein Generator für Beschwerden und Mahnungen.",
+    "featured_image": "stockings-feature",
```
Comment (baltpeter) [inline content/de/blog/v2-release/index.md:6]: The background needs to be a lot darker and bluer for consistency and color contrast.

File: content/de/blog/v2-release/index.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    "authors": [ "zner0L" ]
+}
+
+Gestern haben wir Euch etwas ganz besonderes in den Nikolausstiefel getan: Das neue Release von unserer Webseite mit vielen tollen neuen Features! Wir haben in den letzten paar Monaten viel Zeit damit verbracht, die Nutzer_innenoberfläche zu updaten und haben uns viel Mühe gegeben es so einfach und barrierearm wir möglich zu machen, Anfragen an Unternehmen und anderen Datenverantwortliche zu schicken. Dank des [Prototype Fund](https://prototypefund.de/), der von der [Open Knowledge Foundation Deutschland](https://okfn.de/) betreut und vom [Bundesministerium für Bildung und Forschung](https://bmbf.de) finanziert wird und Lorenz und Benni gefördert hat, konnten wir mehr Zeit als üblich in dieses Release stecken (mehr dazu auf unserer {{< link slug="verein/transparency#12-externe-f%c3%b6rdermittel-die-mitglieder-erhalten-haben" text="Transparenzseite" >}}). Jetzt sind wir sehr aufgeregt und stolz Euch alle neuen Features vorstellen zu können. Wenn Du einen Blick auf den Code werfen möchtest, kannst Du [alle Änderungen auf GitHub finden](https://github.com/datenanfragen/website/pull/921).
```
Comment (baltpeter) [inline content/de/blog/v2-release/index.md:11]:
```suggestion
Gestern haben wir Euch etwas ganz Besonderes in den Nikolausstiefel getan: Das neue Release von unserer Webseite mit vielen tollen neuen Features! Wir haben in den letzten Monaten viel Zeit damit verbracht, die Nutzer_innenoberfläche zu updaten und haben uns viel Mühe gegeben es so einfach und barrierearm wie möglich zu machen, Anfragen an Unternehmen und andere Datenverantwortliche zu schicken. Dank des [Prototype Fund](https://prototypefund.de/), der von der [Open Knowledge Foundation Deutschland](https://okfn.de/) betreut und vom [Bundesministerium für Bildung und Forschung](https://bmbf.de) finanziert wird, und Lorenz und Benni gefördert hat, konnten wir mehr Zeit als üblich in dieses Release stecken (mehr dazu auf unserer {{< link slug="verein/transparency#12-externe-f%c3%b6rdermittel-die-mitglieder-erhalten-haben" text="Transparenzseite" >}}). Jetzt sind wir sehr aufgeregt und stolz, Euch alle neuen Features vorstellen zu können. Wenn Du einen Blick auf den Code werfen möchtest, kannst Du [alle Änderungen auf GitHub finden](https://github.com/datenanfragen/website/pull/921).
```

File: content/de/blog/v2-release/index.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+## Mehrseitiger Anfragengenerator
+
+Wir haben den Generatorablauf komplett neu gestaltet, um ihn aufzuräumen und das Senden vieler Anfragen an viele verschiedene Unternehmen auf einen Schlag zu erleichtern. Während der alte Generator, den wir weiterhin unter {{< link slug="/g" text="/g" >}} zur Verfügung stellen für alle die lieber die alte Oberfläche benutzen wollen, mit einer Seite alles erledigen will, haben wir im neuen Generator den Anfrageprozess auf mehrere Seiten aufgeteilt und begleiten Dich jetzt bis zum Versenden der Anfrage. In der Unternehmensauswahlphase schlagen wir nun außerdem Unternehmenspacks vor, sodass neuen Nutzer_innen der Einstieg erleichtert wird. Im nächsten Schritt kannst Du nun auch auswählen, welche Untereinheiten eines Verantwortlichen Du in Deine Anfrage aufnehmen willst und festlegen, ob es sich um eine Anfrage in Bezug auf Tracking handelt, bei der Du noch nicht weist wie die Verantwortliche dich identifizieren kann.
```
Comment (baltpeter) [inline content/de/blog/v2-release/index.md:15]:
```suggestion
Wir haben den Generatorablauf komplett neu gestaltet, um ihn aufzuräumen und das Senden vieler Anfragen an viele verschiedene Unternehmen auf einen Schlag zu erleichtern. Während der alte Generator, den wir weiterhin unter {{< link slug="/g" text="/g" >}} zur Verfügung stellen für alle, die lieber die alte Oberfläche benutzen wollen, mit einer Seite alles erledigen will, haben wir den Anfrageprozess im neuen Generator auf mehrere Seiten aufgeteilt und begleiten Dich jetzt bis zum Versenden der Anfrage. In der Unternehmensauswahlphase schlagen wir nun außerdem Pakete an Unternehmen vor, sodass neuen Nutzer_innen der Einstieg erleichtert wird. Im nächsten Schritt kannst Du nun auch auswählen, welche Untereinheiten eines Verantwortlichen Du in Deine Anfrage aufnehmen willst und festlegen, ob es sich um eine Anfrage in Bezug auf Tracking handelt, bei der Du noch nicht weißt, wie die Verantwortliche dich identifizieren kann.
```

File: content/de/blog/v2-release/index.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+{{< img name="multi-page-generator-de" alt="Screenshot von drei Seiten des Anfragegenrators (von links nach rechts): Das erste Bild zeigt die Auswahl des Anfragetyps mit den Optionen “Auskunft über meine Daten erhalten.”, “(Teile von) Daten über mich löschen.”, “Daten über mich korrigieren.” and “Widerspruch gegen Direktwerbung einlegen”. Das zweite Bild zeigt die Unternehmensauswahlseite mit einer Suchleiste oben und darunter die Unternehmenspacks mit den Titeln: “Adresshandel und -management”, “Auskunfteien”, “Handel” and “Unterhaltung”. Das dritte Bild zeigt den Identifikationsdatendialog mit einigen Eingabefeldern die nach Identifikationsdaten für den “Datenanfragen.de e. V.“ frageb wie etwa Name, E-Mail-Adresse and Adresse. Unter den Eingabefeldern ist ein Knopf um die Anfrage abzusenden." caption="Drei Seiten des neuen Generators: Die Auswahl des Anfragentyps, die Unternehmenssuche und die Ansicht zum Ausfüllen von Identifikationsdaten." >}}
```
Comment (baltpeter) [inline content/de/blog/v2-release/index.md:17]: The third screenshot is in English.

Comment (baltpeter) [inline content/de/blog/v2-release/index.md:17]:
```suggestion
{{< img name="multi-page-generator-de" alt="Screenshot von drei Seiten des Anfragengenerators (von links nach rechts): Das erste Bild zeigt die Auswahl des Anfragetyps mit den Optionen „Auskunft über meine Daten erhalten.“, „(Teile von) Daten über mich löschen.“, „Daten über mich korrigieren.“ and „Widerspruch gegen Direktwerbung einlegen“. Das zweite Bild zeigt die Unternehmensauswahlseite mit einer Suchleiste oben und darunter die Unternehmenspacks mit den Titeln: „Adresshandel und -management“, „Auskunfteien“, „Handel“ and „Unterhaltung“. Das dritte Bild zeigt den Identifikationsdatendialog mit einigen Eingabefeldern, die nach Identifikationsdaten für den „Datenanfragen.de e. V.“ fragen, wie etwa Name, E-Mail-Adresse und Adresse. Unter den Eingabefeldern ist ein Knopf, um die Anfrage abzusenden." caption="Drei Seiten des neuen Generators: Die Auswahl des Anfragentyps, die Unternehmenssuche und die Ansicht zum Ausfüllen von Identifikationsdaten." >}}
```

File: content/de/blog/v2-release/index.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+
+## Umfangreiche Anfragenhistorie
+
+Wenn Du unsere Webseite regelmäßig nutzt kennst Du vermutlich unsere “Meine Anfragen” Seite. Wenn Du es nicht in den {{< link slug="/privacy-controls" text="Datenschutzeinstellungen" >}} ausgeschaltet hast, speichern wir alle Anfragen, die Du verschickst in Deinem Browser und geben Dir darüber eine Übersicht. Bisher war das nur eine unübersichtliche Tabelle, die wir schon seit Jahren verbessern und als richtiges Anfragenverwaltungstool aufbauen wollen. In unserem neuen Release erstellen wir jetzt einen Vorgang für jede Deine Anfragen, der alle Nachrichten enthält, die etwas mit ihr zu tun haben einschließlich Mahnungen und Beschwerden. Du kannst Informationen über Nachrichten, die Du von der Verantwortlichen erhälst, hinzufügen, sodass du alles an einem Ort gesammelt hast, falls Du dann eine Beschwerde senden möchtest. Wir speichern den Status all Deiner Anfragen, damit Du ihren aktuellen Stand verfolgen kannst, und erinnern Dich mit einer Badge auf allen Seiten daran, falls es überfällige Anfragen gibt für die Du eine Mahnung verschicken solltest.
```
Comment (baltpeter) [inline content/de/blog/v2-release/index.md:21]:
```suggestion
Wenn Du unsere Webseite regelmäßig nutzt, kennst Du vermutlich unsere „Meine Anfragen“-Seite. Wenn Du es nicht in den {{< link slug="/privacy-controls" text="Datenschutzeinstellungen" >}} ausgeschaltet hast, speichern wir alle Anfragen, die Du verschickst, in Deinem Browser und geben Dir darüber eine Übersicht. Bisher war das nur eine unübersichtliche Tabelle, die wir schon seit Jahren verbessern und als richtiges Anfragenverwaltungstool ausbauen wollten. In unserem neuen Release erstellen wir jetzt einen Vorgang für jede Deine Anfragen, der alle Nachrichten enthält, die etwas mit ihr zu tun haben, einschließlich Mahnungen und Beschwerden. Du kannst Informationen über Nachrichten, die Du von der Verantwortlichen erhältst, hinzufügen, sodass Du alles an einem Ort gesammelt hast, falls Du dann eine Beschwerde senden möchtest. Wir speichern den Status all Deiner Anfragen, damit Du ihren aktuellen Stand verfolgen kannst, und erinnern Dich auf allen Seiten daran, falls es überfällige Anfragen gibt, für die Du eine Mahnung verschicken solltest.
```

File: content/de/blog/v2-release/index.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+
+{{< img name="my-requests" alt="Screenshot der “Meine Anfragen” Seite. Unter dem Erklärungstext stehen mehrere Vorgänge mit verschiedenen Status, wie etwa “wartet auf Antwort”, “überfällig” and “abgeschlossen”. Ein Vorgang ist aufgeklappt und enthält zwei Nachrichten, eine von der Nutzer_in und eine des angefragten Unternehmens." caption="Die neue Anfragenhistorie auf der “Meine Anfragen“ Seite." >}}
```
Comment (baltpeter) [inline content/de/blog/v2-release/index.md:23]:
```suggestion
{{< img name="my-requests" alt="Screenshot der „Meine Anfragen“-Seite. Unter dem Erklärungstext stehen mehrere Vorgänge mit verschiedenen Status, wie etwa „wartet auf Antwort“, „überfällig“ und „abgeschlossen“. Ein Vorgang ist aufgeklappt und enthält zwei Nachrichten, eine von der Nutzer_in und eine des angefragten Unternehmens." caption="Die neue Anfragenhistorie auf der „Meine Anfragen“-Seite." >}}
```

**Review by baltpeter — CHANGES_REQUESTED** [review]

File: content/de/blog/v2-release/index.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+
+## Beschwerdegenerator und Anfragereaktionen
+
+Die neuen “Meine Anfragen” Seite kommt erst mit einem weiteren Feature so richtig zur Geltung, auf das wir besonders stolz sind: Basierend auf Deiner Anfragenhistorie geben wir Dir die Möglichkeit zu “reagieren”. Wenn Du Dich dazu entscheidest, helfen wir Dir einzuschätzen, welche Fehler eine Verantwortliche möglicherweise gemacht hat, indem wir Dir ein paar Fragen stellen. Daraus generieren wir dann eine Mahnung oder sogar eine richtige Beschwerde. Auch wenn wir nochmal deutlich machen wollen, dass wir keine Anwälte sind und keine Rechtsberatung für spezifische Fälle anbieten können, hoffen wir trotzdem, dass diese Feature es Dir leichter macht, Dich bei den Aufsichtsbehörden zu beschwerden. Du bist mit Deiner Überforderung von Unternehmensantworten nicht mehr allein.
```
Comment (baltpeter) [inline content/de/blog/v2-release/index.md:27]:
```suggestion
Die neuen „Meine Anfragen“-Seite kommt erst mit einem weiteren Feature so richtig zur Geltung, auf das wir besonders stolz sind: Basierend auf Deiner Anfragenhistorie geben wir Dir die Möglichkeit zu „reagieren“. Wenn Du Dich dazu entscheidest, helfen wir Dir einzuschätzen, welche Fehler eine Verantwortliche möglicherweise gemacht hat, indem wir Dir ein paar Fragen stellen. Daraus generieren wir dann eine Mahnung oder sogar eine richtige Beschwerde. Auch wenn wir nochmal deutlich machen wollen, dass wir keine Anwälte sind und keine Rechtsberatung für spezifische Fälle anbieten können, hoffen wir trotzdem, dass diese Feature es Dir leichter macht, Dich bei den Aufsichtsbehörden zu beschweren. Du bist mit Deiner Überforderung von Unternehmensantworten nicht mehr allein.
```

Comment (baltpeter) [reply content/de/blog/v2-release/index.md:27]: I already didn't particularly like the English version but I doubt many people are going to understand "Du bist mit Deiner Überforderung von Unternehmensantworten nicht mehr allein.".

File: content/de/blog/v2-release/index.md [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+
+{{< img name="complaint-generator" alt="Screenshot des Mahnungsgenerators. Ganz oben steht die Frage “Gab es ein Problem mit Deiner Anfrage?” und darunter verschiedene Buttons für verschiedene Arten von Problemen. Ein Beispiel für einen Buttontext: “Das Unternehmen fragt nach zusätzlichen Daten zur Authentifizierung.”" caption="Erste Seite des Mahnungsgenerators." >}}
```
Comment (baltpeter) [inline content/de/blog/v2-release/index.md:29]:
```suggestion
{{< img name="complaint-generator" alt="Screenshot des Mahnungsgenerators. Ganz oben steht die Frage „Gab es ein Problem mit Deiner Anfrage?“ und darunter verschiedene Buttons für verschiedene Arten von Problemen. Ein Beispiel für einen Buttontext: „Das Unternehmen fragt nach zusätzlichen Daten zur Authentifizierung.“" caption="Erste Seite des Mahnungsgenerators." >}}
```

File: content/de/blog/v2-release/index.md [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+
+## Schicke Startseite
+
+Eine weitere Sache, die schon seit Jahren auf unserer Wunschliste steht ist die Verbesserung underer Startseite, die nie wirklich gut zu dem gepasst hat, was wir eigentlich machen. Jetzt leiten wir Nutzer_innen zum Anfragengenerator mit klaren Calls-to-action und geben einen kurzen Überblick wie es funktioniert. Selbst für erfahrene Nutzer_innen lohnt sich hin und wieder der Besuch der Startseite: Du kannst dort dank unseres Erinnerungswidgets direkt auf überfällige Anfragen reagieren und außerdem finden zeigen wir dort unsere neusten Blogartikel.
```
Comment (baltpeter) [inline content/de/blog/v2-release/index.md:33]:
```suggestion
Eine weitere Sache, die schon seit Jahren auf unserer Wunschliste steht, ist die Verbesserung unserer Startseite, die nie wirklich gut zu dem gepasst hat, was wir eigentlich machen. Jetzt leiten wir Nutzer_innen zum Anfragengenerator mit klaren Calls-to-action und geben einen kurzen Überblick, wie der Prozess funktioniert. Selbst für erfahrene Nutzer_innen lohnt sich hin und wieder der Besuch der Startseite: Du kannst dort dank unseres Erinnerungswidgets direkt auf überfällige Anfragen reagieren und außerdem zeigen wir dort unsere neusten Blogartikel.
```

File: content/de/blog/v2-release/index.md [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+
+{{< img name="homepage" alt="Screenshot der Startseite. Sie hat eine Kopfleiste mit dem Menü und einer Badge, die auf die überfälligen Anfragen hinweist (14) und darunter ein Widget mit dem Text: “Du hast 14 überfällige Anfragen. Reagiere jetzt oder markiere sie als zufriedenstellend.”. Darunter steht groß der Satz: “Du hast ein Recht auf Vergessenwerden” und daneben sind vier Buttons für die Anfragetypen." caption="Unsere neue Startseite mit Erinnerungswidget." >}}
```
Comment (baltpeter) [inline content/de/blog/v2-release/index.md:35]:
```suggestion
{{< img name="homepage" alt="Screenshot der Startseite. Sie hat eine Kopfleiste mit dem Menü und einer Badge, die auf die überfälligen Anfragen hinweist (14) und darunter ein Widget mit dem Text: „Du hast 14 überfällige Anfragen. Reagiere jetzt oder markiere sie als zufriedenstellend.“. Darunter steht groß der Satz: „Du hast ein Recht auf Vergessenwerden.“ und daneben sind vier Buttons für die Anfragetypen." caption="Unsere neue Startseite mit Erinnerungswidget." >}}
```

File: content/de/blog/v2-release/index.md [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+
+## Refactoring: TypeScript, zustand und weitere Verbesserungen
+
+Im Maschinenraum (im Code) haben wir auch viel verändert. Manche der Änderungen haben wir auch schon im laufe der letzten Monate veröffentlicht. Wir sind zu [TypeScript](https://www.typescriptlang.org/) gewechselt um Fehler früher zu entdecken (und Mädchen haben wir viele gefunden) und nutzen jetzt [zustand](https://github.com/pmndrs/zustand) für Statemanagement, in der Hoffnung, dass das Race condition bugs im Code behebt. Das heißt auch, dass wir die alten gespeicherten Anfragen in eine neue Speicherart überführt haben. Wir hoffen, dass alles gut gegangen ist, aber falls Du irgendwelche Probleme bemerkt hast und/oder irgendwelche Anfragen vermisst, [schreib uns eine E-Mail](mailto:dev@datenanfragen.de). Außerdem haben wir den Code ordentlich aufgeräumt und seine alten und disfunktionalen Ecken entfernt. Und wir veröffentlichen jetzt [ein Package unserer Generators](https://www.npmjs.com/package/@datenanfragen/components), dass Du in Deinen eigenen Code einbauen kannst, z.B. wenn du es in Deine Webseite einbetten willst.
```
Comment (baltpeter) [inline content/de/blog/v2-release/index.md:39]:
```suggestion
Im Maschinenraum (im Code) haben wir auch viel verändert. Manche der Änderungen haben wir auch schon im Laufe der letzten Monate veröffentlicht. Wir sind zu [TypeScript](https://www.typescriptlang.org/) gewechselt, um Fehler früher zu entdecken (und Mädchen haben wir viele gefunden) und nutzen jetzt [zustand](https://github.com/pmndrs/zustand) für das Statemanagement, in der Hoffnung, dass das Race-Condition-Bugs im Code behebt. Das heißt auch, dass wir die alten gespeicherten Anfragen in eine neue Speicherart überführt haben. Wir hoffen, dass alles gut gegangen ist, aber falls Du irgendwelche Probleme bemerkt hast und/oder irgendwelche Anfragen vermisst, [schreib uns eine E-Mail](mailto:dev@datenanfragen.de). Außerdem haben wir den Code ordentlich aufgeräumt und seine alten und dysfunktionalen Ecken entfernt. Und wir veröffentlichen jetzt [ein Package unserer Generators](https://www.npmjs.com/package/@datenanfragen/components), das Du in Deinen eigenen Code einbauen kannst, z.&nbsp;B. wenn du es in Deine Webseite einbetten willst.
```

File: content/de/blog/v2-release/index.md [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+
+## Hilfe gesucht: Fehlermeldungen, Unternehmensvorschläge und Übersetzungen
+
+Jetzt da wir die neuen Features veröffentlicht haben, brauchen wir eure Hilfe: Wir haben versucht alles intensiv zu testen, aber manchmal schlüpft eben doch ein Fehler druch. Falls Du also einen findest, schick uns gerne eine Fehlermeldung, entweder über unsere Fehlermeldungsfenster oder an [dev@datenanfragen.de](mailto:dev@datenanfragen.de). Außerdem haben sich viele Texte verändert oder wurden neu in unsere Übersetzungen hinzugefügt. Wenn Du mitmachen willst, kannst Du sie [via Weblate](https://hosted.weblate.org/projects/datenanfragen-de/website/) in andere Sprachen übersetzen, das wäre wahnsinnig toll! Schlißelich können wir nur in den Länder Unternehmen vorschlagen, für die wir auch Unternehmenspacks haben. Für Deustchland haben wir einen guten Überblick, aber falls Du denkst, dass Du für ein anderes Land gut weißt, welche Unternehmen regelmäßig die Daten von vielen Menschen in diesem Land verarbeiten, füg gerne ein Unternehmenspack [in unsere Datenbank](https://github.com/datenanfragen/data#company-packs) hinzu (Bitte gibt dabei ausführliche Quellen an).
```
Comment (baltpeter) [inline content/de/blog/v2-release/index.md:43]:
```suggestion
Jetzt, da wir die neuen Features veröffentlicht haben, brauchen wir eure Hilfe: Wir haben versucht alles intensiv zu testen, aber manchmal schlüpft eben doch ein Fehler durch. Falls Du also einen findest, schick uns gerne eine Fehlermeldung, entweder über unsere Fehlermeldungsfenster oder an [dev@datenanfragen.de](mailto:dev@datenanfragen.de). Außerdem haben sich viele Texte verändert oder wurden neu in unsere Übersetzungen hinzugefügt. Wenn Du mitmachen willst, kannst Du sie [via Weblate](https://hosted.weblate.org/projects/datenanfragen-de/website/) in andere Sprachen übersetzen, das wäre unfassbar toll! Schließlich können wir nur in den Länder Unternehmen vorschlagen, für die wir auch Unternehmenspacks haben. Für Deutschland haben wir einen guten Überblick, aber falls Du denkst, dass Du für ein anderes Land gut weißt, welche Unternehmen regelmäßig die Daten von vielen Menschen in diesem Land verarbeiten, füg gerne ein Unternehmenspack [in unsere Datenbank](https://github.com/datenanfragen/data#company-packs) hinzu (bitte gibt dabei ausführliche Quellen an).
```

File: content/de/blog/v2-release/index.md [hunk 18]
```diff
@@ -0,0 +1,43 @@
+{
+    "title": "Datenanfragen.de 2.0: Nagelneue Oberfläche und Beschwerdegenerator",
+    "type": "blog",
+    "date": "2022-12-07T12:00:00+02:00",
```
Comment (baltpeter) [inline content/de/blog/v2-release/index.md:4]: The slug should be translated accordingly.

File: content/de/blog/v2-release/index.md [hunk 19]
```diff
@@ -0,0 +1,43 @@
...
+    "authors": [ "zner0L" ]
+}
+
+Gestern haben wir Euch etwas ganz Besonderes in den Nikolausstiefel getan: Das neue Release von unserer Webseite mit vielen tollen neuen Features! Wir haben in den letzten Monaten viel Zeit damit verbracht, die Nutzer_innenoberfläche zu updaten und haben uns viel Mühe gegeben es so einfach und barrierearm wie möglich zu machen, Anfragen an Unternehmen und andere Datenverantwortliche zu schicken. Dank des [Prototype Fund](https://prototypefund.de/), der von der [Open Knowledge Foundation Deutschland](https://okfn.de/) betreut und vom [Bundesministerium für Bildung und Forschung](https://bmbf.de) finanziert wird, und Lorenz und Benni gefördert hat, konnten wir mehr Zeit als üblich in dieses Release stecken (mehr dazu auf unserer {{< link slug="verein/transparency#12-externe-f%c3%b6rdermittel-die-mitglieder-erhalten-haben" text="Transparenzseite" >}}). Jetzt sind wir sehr aufgeregt und stolz, Euch alle neuen Features vorstellen zu können. Wenn Du einen Blick auf den Code werfen möchtest, kannst Du [alle Änderungen auf GitHub finden](https://github.com/datenanfragen/website/pull/921).
+
```
Comment (baltpeter) [inline content/de/blog/v2-release/index.md:12]: Any reason you're not including the featured image here like in all other posts (also for EN).

File: content/en/blog/v2-release/index.md [hunk 20]
```diff
@@ -0,0 +1,43 @@
...
+
+{{< img name="multi-page-generator-en" alt="Screenshot of three pages of the multi-page-generator (from left to right): the first image shows the request type choosing dialog containg the options “Get access to data about me.”, “Delete (parts of) data about me.”, “Correct data about me.” and “Stop receiving direct marketing”. The second image shows the company choosing page with a search bar on top and several suggestion packs under it, with the titles: “Social media and communication”, “Entertainment”, “Finance” and “Telecommunication”. The third image show the identification data dialog with several input fields asking for identification data for “Datenanfragen.de e. V.“ such as name, email address and address. Under the input fields there is a button to send the request." caption="Three pages of the new generator: The request type chooser, the company search and the view for filling in you identifcation data." >}}
```
Comment (baltpeter) [inline content/en/blog/v2-release/index.md:17]:
```suggestion
{{< img name="multi-page-generator-en" alt="Screenshot of three pages of the multi-page-generator (from left to right): the first image shows the request type choosing page listing the options “Get access to data about me.”, “Delete (parts of) data about me.”, “Correct data about me.” and “Stop receiving direct marketing”. The second image shows the company choosing page with a search bar on top and several suggested company packs under it, with the titles: “Social media and communication”, “Entertainment”, “Finance” and “Telecommunication”. The third image show the identification data page with several input fields asking for identification data for “Datenanfragen.de e. V.“ such as name, email address and address. Under the input fields, there is a button to send the request." caption="Three pages of the new generator: The request type chooser, the company search and the view for filling in your identification data." >}}
```

File: content/en/blog/v2-release/index.md [hunk 21]
```diff
@@ -0,0 +1,43 @@
...
+
+{{< img name="my-requests" alt="Screenshot of the “My requests”-page. Under the explanation text there are several requests with different states, such as “Action needed”, “Overdue” and “Done”. One request is folded out showing two messages, one from the user and one from the requested company." caption="The new request history on the “My requests“-page." >}}
```
Comment (baltpeter) [inline content/en/blog/v2-release/index.md:23]:
```suggestion
{{< img name="my-requests" alt="Screenshot of the “My requests” page. Under the explanation text, there are several requests with different states, such as “Action needed”, “Overdue” and “Done”. One request is expanded showing two messages, one from the user and one from the requested company." caption="The new request history on the “My requests“ page." >}}
```

File: content/en/blog/v2-release/index.md [hunk 22]
```diff
@@ -0,0 +1,43 @@
...
+
+## Complaint generator & request reactions
+
+The new “My requests” page shines even more when used in combination with the feature we are most proud of: Based on the history of your request, we offer you the option to “react”. If you choose to do so, we help you assess what missteps a controller might have taken by asking you some questions and then generate an admonition or even a proper complaint from your answers. While we want to be clear that we are no lawyers and can't offer help in specific cases, we do hope this makes it easier for you to actually complain to the authorities. You are not alone with being overwhelmed by controller replies anymore.
+
+{{< img name="complaint-generator" alt="Screenshot of the admonition generator. On top, it shows the question “Has there been a problem with your request?” and below that are several buttons featuring kinds of problems. One example of a button text: “Company has not answered yet.”" caption="First page of the admonition generator" >}}
```
Comment (baltpeter) [inline content/en/blog/v2-release/index.md:29]:
```suggestion
{{< img name="complaint-generator" alt="Screenshot of the admonition generator. On top, it shows the question “Has there been a problem with your request?” and below that are several buttons featuring different kinds of problems. One example of a button text: “Company has not answered yet.”" caption="First page of the admonition generator" >}}
```

**Review by baltpeter — CHANGES_REQUESTED** [review]

File: content/de/blog/v2-release/index.md (hunk 18 above)
Comment (zner0L) [reply content/de/blog/v2-release/index.md:4]: What do you mean?

Comment (baltpeter) [reply content/de/blog/v2-release/index.md:4]: Use something other than `v2-release` for the slug.

Comment (zner0L) [reply content/de/blog/v2-release/index.md]: Why?

Comment (zner0L) [reply content/de/blog/v2-release/index.md]: But this is perfectly fine in both languages?

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**